src/codeu/chat/client/core/Controller.java

```diff
@@ -112,4 +114,27 @@ public ConversationHeader newConversation(String title, Uuid owner)  {
 
     return response;
   }
+  
+  public Time statusUpdate() {
+	Time updateTime = null;
+	try (final Connection connection = source.connect()) {
+
+	  Serializers.INTEGER.write(connection.out(), NetworkCode.STATUS_UPDATE_REQUEST);
+	  
```

Comment:
might need to write an updated conversationInterests Hashmap to update the conversationInterests object of the model's users.